photon-client/src/types/PhotonTrackingTypes.ts

```diff
@@ -0,0 +1,29 @@
+export interface Pose {
+    x: number,
+    y: number,
+    z: number,
+    angle_z: number,
+    qw: number,
+    qx: number,
+    qy: number,
+    qz: number
+}
+
+export interface PhotonTarget {
+    yaw: number,
+    pitch: number,
+    skew: number,
+    area: number,
+    // -1 if not set
+    ambiguity: number,
```

Comment:
For the things in photon-tracking, I just want to indicate that they would always be present (with a negative one value if non-tag pipeline). I don't think a default would make sense there because there isn't a default ambiguity or fiducial Id. The only place where defaults would be good is in the pipeline and camera settings but we already handle that with the placeholder settings.